critiquebrainz/ws/review/test/views_test.py

```diff
@@ -186,3 +190,33 @@ def test_revision_entity_handler(self):
         self.client.post('/review/%s' % review["id"], headers=self.header(self.user), data=json.dumps(data))
         resp = self.client.get('/review/%s/revisions/2' % review["id"])
         self.assert200(resp)
+
+    def test_cache_tracking(self):
+        entity_id = self.review["entity_id"]
+        track_key = cache.gen_key("ws_cache", entity_id)
+
+        # Test no cache if entity id is not provided
+        self.client.get('/review/', query_string={'sort': 'rating'})
+        cache_keys = cache.get(track_key, namespace="Review")
+        self.assertEqual(None, cache_keys)
+
+        expected_cache_keys = [b'list_6b3cd75d-7453-39f3-86c4-1441f360e121_None_popularity_50_0_None',
```

Comment:
can we have at least one test that checks all of the items in this key (I'm looking at the two Nones)